source/MaterialXView/Main.cpp

```diff
@@ -245,12 +263,22 @@ int main(int argc, char* const argv[])
         viewer->setEnvSampleCount(envSampleCount);
         viewer->setLightRotation(lightRotation);
         viewer->setDocumentModifiers(modifiers);
+        viewer->setBakeWidth(bakeWidth);
+        viewer->setBakeHeight(bakeHeight);
+        viewer->setBakeFilename(bakeFilename);
+        viewer->initialize();
+        if (!bakeFilename.empty()) 
+        {
+            viewer->bakeTextures();
+            viewer->requestExit();
+        } else {            
```

Comment:
For consistency with coding conventions, these two brackets should both be on their own line.